src/components/content/content.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import './content.scss';
+import SurveyHeader from "./survey-header/survey-header";
+import SurveyTable from "./survey-table/survey-table";
+import SurveyTableHeader from "./survey-table/survey-table-header/survey-table-header";
+import SurveyTableRow from "./survey-table/survey-table-row/survey-table-row";
+import store from "../../store";
+import SurveyCheckbox from "../shared/survey-checkbox/survey-checkbox";
+import {checkAllSurveyTableRows} from "../../store/actions";
+
+
+class ContentComponent extends React.Component {
+    state = {
+        headers: [],
+        sorted: {},
+        surveys: []
+    };
+
+    checkAll = false;
+
+    componentDidMount() {
+        this.setState(store.getState());
+        store.subscribe(
+            () => {
+                const newState = store.getState();
+                this.checkAll = newState.surveys.find(s => s.checked === false) === undefined;
+                this.setState(newState);
+            }
+        )
+    }
+
+    onAllCheckboxSelect = () => {
+        store.dispatch(checkAllSurveyTableRows());
+    };
+
+    render() {
+        const {sorted, headers, surveys} = this.state;
+        return (
+            <div className="col-11 content pb-5">
+                <div className="row justify-content-center">
+                    <SurveyHeader/>
```

Comment:
This API doesn't make sense to me. Why table header is outside of the table? Why there are two tables? Why do you mapping over every survey and rendering table for each one?
Check for example API of [Segments `Table` component](https://evergreen.segment.com/components/table/). If you check any other UI library with Table component you'll very similar solutions.